.kokoro/release/stage-non-google-oss-sonatype.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# This stage-non-google-oss-sonatype.sh publishes the artifacts to oss.sonatype.org publishing
+# host, not google.oss.sonatype.org. A group ID of Maven artifact can belong to
+# one publishing host. As of February 2023, com.google.maps group
+# ID is not part of the google.oss.sonatype.org publishing host. In future when
+# we move com.google.maps (and others) to the
+# google.oss.sonatype.org publishing host, we can stop this job separated.
+
+# Start the releasetool reporter
+requirementsFile=$(realpath $(dirname "$0")/../../)/.kokoro/requirements.txt
+python3 -m pip install --require-hashes -r $requirementsFile
+python3 -m releasetool publish-reporter-script > /tmp/publisher-script; source /tmp/publisher-script
+
+source $(dirname "$0")/common.sh
+MAVEN_SETTINGS_FILE=$(realpath $(dirname "$0")/../../)/settings.xml
+pushd $(dirname "$0")/../../
+
+setup_environment_secrets
+create_settings_xml_file "settings.xml"
+
+echo "Deploying non-Cloud artifacts to staging repositories in oss.sonatype.org"
+
+# Lists modules for Google Maps artifacts, including their submodules.
+includedMapsModule=$(find java-maps-* -name 'pom.xml'  \
```

Comment:
Should we change `includedMapsModule` to `nonCloudModules`? Maybe there will be artifacts similar to this one.

Reply:
Good point. Renaming.